content/integrate/redis-data-integration/data-pipelines/deploy.md

```diff
@@ -78,72 +78,44 @@ The specific command lines for source secrets are as follows:
 
 ```bash
 # Source username
-kubectl create secret generic source-db \
---namespace=rdi \
---from-literal=SOURCE_DB_USERNAME=yourUsername
-
-# Source password
-kubectl create secret generic source-db \
---namespace=rdi \
---from-literal=SOURCE_DB_PASSWORD=yourPassword
-
-# Source trust certificate (both commands are required)
-kubectl create secret generic source-db-ssl --from-file=ca.crt=/path/to/myca.crt -n rdi
-
-kubectl create secret generic source-db \
---namespace=rdi \
---from-literal=SOURCE_DB_CACERT=/etc/certificates/source_db/ca.crt
-
-# Source public key (both commands are required)
-kubectl create secret generic source-db-ssl --from-file=client.crt=/path/to/myclient.crt -n rdi
-
-kubectl create secret generic source-db \
---namespace=rdi \
---from-literal=SOURCE_DB_CERT=/etc/certificates/source_db/client.crt
-
-
-# Source private key (both commands are required)
-kubectl create secret generic source-db-ssl --from-file=client.key=/path/to/myclient.key -n rdi
-
-kubectl create secret generic source-db \
---namespace=rdi \
---from-literal=SOURCE_DB_KEY=/etc/certificates/source_db/client.key
+kubectl create secret generic source-db --namespace=rdi \
+--from-literal=SOURCE_DB_USERNAME=yourUsername \
+--from-literal=SOURCE_DB_PASSWORD=yourPassword \
```

Comment:
The 3 keys that are already mentioned in the ticket:
```
--from-literal=SOURCE_DB_CACERT=/etc/certificates/source_db/ca.crt \
--from-literal=SOURCE_DB_CERT=/etc/certificates/source_db/client.crt \
--from-literal=SOURCE_DB_KEY=/etc/certificates/source_db/client.key \
```
are really required as well if TLS or mTLS is enabled, in the same secret `source-db`. They may be removed in the future, but for now they are required and should be specified:
* Only `SOURCE_DB_CACERT` for TLS
* All 3 of them for mTLS

The same is true for the target. 

Note that this command recreates the secret, meaning that it should contain all needed keys, not just a subset, and therefore it can't be split into multiple commands. Instead, we could perhaps have the `Source TLS` and `Source mTLS` sections below contain both commands - the one that creates `source-db` with all needed keys, and the one that creates `source-db-ssl`.

Reply:
Also, the key `SOURCE_DB_KEY_PASSWORD` may be required for the mTLS case if the key is password-protected. To avoid writing yet another set of commands just for that, I would suggest to add it to the mTLS commands with a comment, something like:
```
--from-literal=SOURCE_DB_KEY_PASSWORD=yourKeyPassword \ # add this only if SOURCE_DB_KEY is password-protected
```

The same is true for the target.

Reply:
I'm sorry, @stoyanr, but I'm not clear on what you want me to change. @yaronp68: feel free to weigh in.

Reply:
Resolved in next push.